app/src/main/java/org/mozilla/fenix/tabstray/browser/BrowserTrayList.kt

```diff
@@ -63,7 +63,7 @@ class BrowserTrayList @JvmOverloads constructor(
         TabsTouchHelper(
             observable = adapter as TabsAdapter,
             onViewHolderTouched = { swipeToDelete.isSwipeable },
-            onViewHolderDraw = { context.components.settings.listTabView }
+            onViewHolderDraw = { context.components.settings.gridTabView.not() }
```

Comment:
How come we prefer `.not()` compare to `!`?

Reply:
With the long line, it was too easy for myself to not see the `!` at one end. Figured it might make it easier for the next person to see the "gridTabView" and "not" together so it clicks.

Reply:
That's fair. This was more of a curiosity thing since I am seeing more `.not()`.